packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -380,6 +382,76 @@ const renderPointGraphControls = (props: {
     );
 };
 
+/**
+ * TODO [catjohnson]: The polygon controls will be vastly different from the point controls.
+ * Will be keeping this function as is for the time being, even if it's a copy of the point graph
+ * controls.
+ */
+const renderPolygonGraphControls = (props: {
+    state: PolygonGraphState;
+    dispatch: (action: InteractiveGraphAction) => unknown;
+    width: number;
+    strings: PerseusStrings;
```

Comment:
should `state`, `dispatch`, `width`, and `strings` be its own type? Or is there an existing type that could be destructured? 

It would be similar to what you have now
```suggestion 
const renderPolygonGraphControls( {
    state,
    dispatch,
    width,
    strings
}: GraphControls)
```

Then you could use the params outright without using `props.<param>`

Reply:
I don't think there is an existing type that works for this function, and personally I'm not for creating new types for function parameters (I only do it for components and classes).

Let me know if you disagree or have other thoughts 😉